File: 09_Clean Code/praktikum/Rewrite.js
class kendaraan {
    var totalroda = 0;
    var kecepatanperjam = 0;
}

class mobil extends kendaraan {
    void bergerak(){
        tambahkecepatan(10);
    }
        
    tambahkecepatan(var kecepatanbaru){
        kecepatanperjam = kecepatanperjam + kecepatanbaru;
    }
}

void main(){
    mobilcepat = new mobil();
    while (ulang < 4) {
        mobilcepat.bergerak();
        ulang++;
      }

    mobillamban = new mobil();
    mobillamban.bergerak();
}

File: 09_Clean Code/praktikum/Rewrite.go
package main

type kendaraan struct{
	totalroda int
	kecepatanperjam int
}

type mobil struct{
	kendaraan
}

func (m *mobil) bergerak(){
	m.tambahkecepatan(10)
}

func (m *mobil) tambahkecepatan(kecepatanbaru int){
	m.kecepatanperjam = m.kecepatanperjam + kecepatanbaru
}

func main(){
	mobilcepat := mobil{}
	mobilcepet.bergerak()
	mobilcepet.bergerak()
	mobilcepet.bergerak()

	mobillambat := mobil{}
	mobillambat.bergerak
}